### PR TITLE
SPORTSHUB-182 fixed share modal on the wrong pages

### PR DESCRIPTION
**Pull request by ReggieStar11**
## Why was your change needed?

Please detail here why you did the change you did.

## What was Changed?

Please detail here what were the main changes you made.

## Any extra notes for reviewers to know?

Write any extra notes you may have for the PR.
This may include specific code paths that need extra scrutiny, or how to review a specific portion of code.

### CODE REVIEW

**Comment by owya490** [comment]
we have a few conflicts here?

**Review by owya490 — APPROVED** [review]
so this just move it to the bottom?

File: frontend/app/(no-footer)/organiser/event/[id]/page.tsx [hunk 1]
```diff
@@ -121,9 +121,10 @@ export default function EventPage({ params }: EventPageProps) {
             />
           </div>
           <div id="event-drilldown-details-page" className="w-full">
-            <ShareModal/>
+            {/* <ShareModal/> */}
```
[the remark this answers is not in the document]
Comment (ReggieStar11) [reply frontend/app/(no-footer)/organiser/event/[id]/page.tsx]: yep

**Comment by ReggieStar11** [comment]
> so this just move it to the bottom?

yea and then only have it on the event details

**Review by owya490 — APPROVED** [review]
nice lgtm

**Pull request merged**